FWCore/SharedMemory/interface/ReadBuffer.h

```diff
@@ -26,18 +26,19 @@ This works in conjunction with WriteBuffer.
 
 // user include files
 #include "FWCore/SharedMemory/interface/buffer_names.h"
+#include "FWCore/SharedMemory/interface/BufferInfo.h"
 
 // forward declarations
 
 namespace edm::shared_memory {
   class ReadBuffer {
   public:
     /** iUniqueName : must be unique for all processes running on a system.
-        iBufferIndex : is a pointer to a shared_memory address where the same address needs to be shared by ReadBuffer and WriteBuffer.
+        iBufferInfo : is a pointer to a shared_memory address where the same address needs to be shared by ReadBuffer and WriteBuffer.
```

Comment:
Have you kept the same description on purpose?

Reply:
Yes

Reply:
The purpose of the variable is unchanged, I just changed the name to better reflect what it was doing.